src/components/common/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,12 @@
+import { FC } from "react";
+import { P1 } from "@deskpro/deskpro-ui";
+
+export type Props = {
+  text?: string,
+};
+
+const NotFound: FC<Props> = ({ text = "No found" } = {}) => (
+  <P1>{text}</P1>
+);
+
```

Comment:
Minor: Would be better to use `children: ReactNode` here instead. More flexiable.

Reply:
I'll fix it in next PR